src/components/AppFooter/AppFooter.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const AppFooter: React.FC = () => {
```

Comment:
Her kan 
const AppFooter = () => { 
brukes isteden.

Se: [link](https://www.harrymt.com/blog/2020/05/20/react-typescript-react-fc.html) for reasoning.

Ellers ser det bra ut!